dbt/include/trino/macros/adapters.sql

```diff
@@ -89,8 +89,12 @@
 {%- endmacro -%}
 
 {% macro comment(comment) %}
-  {%- if comment is not none and comment|length > 0 -%}
-      comment '{{ comment | replace("'", "''") }}'
+  {%- set persist_docs = model['config'].get('persist_docs', {}) -%}
+  {%- if persist_docs is not none -%}
```

Comment:
```suggestion
  {%- if persist_docs -%}
```
We should check if `persist_docs` evaluates to True. Currently it checks if it is not None
(`{}` will pass current condition, but it shouldn't)